src/content/reference/react-dom/server/renderToString.md

```diff
@@ -4,13 +4,13 @@ title: renderToString
 
 <Pitfall>
 
-`renderToString` does not support streaming or waiting for data. [See the alternatives.](#alternatives)
+`renderToString` tidak mendukung streaming atau menunggu data. [Lihat alternatif.](#alternatives)
```

Comment:
```suggestion
`renderToString` tidak mendukung *streaming* atau menunggu data. [Lihat beberapa alternatifnya.](#alternatives)
```

Reply:
Please don't forget to italicize the `streaming` word, @hisamafahri. Thanks!

Reply:
Ah, missed that one. Should be fixed now